sqlite/ext/misc/completion.c

```diff
@@ -242,10 +242,10 @@ static int completionNext(sqlite3_vtab_cursor *cur){
       case COMPLETION_TABLES: {
         if( pCur->pStmt==0 ){
 #if defined(SQLITE_BUILDING_FOR_COMDB2)
-          /* TODO: UNION vtables? */
           sqlite3_prepare_v2(pCur->db, 
                   "SELECT tablename FROM comdb2_tables "
-                  "WHERE tablename NOT LIKE 'sqlite_stat%' ORDER BY 1", 
+                  "WHERE tablename NOT LIKE 'sqlite_stat%' UNION "
+                  "SELECT name FROM comdb2_systables ORDER BY 1", 
```

Comment:
I gave it a shot but for some reason auto-completion did not return system table names.

Reply:
I assume you ran the correct cdb2sql exe. 
It should show after something like 'select * from ^I'  (ie not as the first level token). Ill try to see if it works for me in a fresh checkout/build.

Reply:
Seems to work for me, try 'select * FROM comdb2_col^I'

Reply:
Remember, you have to restart the server with the new comdb2 executable to get the autocomplete list from the server itself.

Reply:
ok, its working now.

Reply:
I think, I used a wrong binary.